packages/components/fields/async-creatable-select-field/src/async-creatable-select-field.stories.tsx

```diff
@@ -0,0 +1,193 @@
+import type { Meta, StoryFn } from '@storybook/react';
+import AsyncCreatableSelectField from './async-creatable-select-field';
+import { useEffect, useState } from 'react';
+import { iconArgType } from '@/storybook-helpers';
+
+const meta: Meta<typeof AsyncCreatableSelectField> = {
+  title: 'Form/Fields/AsyncCreatableSelectField',
+  // @ts-expect-error, @todo: fix typings of field component
+  component: AsyncCreatableSelectField,
+  argTypes: {
+    // @ts-expect-error, @todo: fix typings of field component
+    hintIcon: iconArgType,
+    iconLeft: iconArgType,
+    id: { control: { type: 'text' } },
+    isMulti: { control: { type: 'boolean' } },
+    hint: { control: { type: 'text' } },
+    description: { control: { type: 'text' } },
+    badge: { control: { type: 'text' } },
+    name: { control: { type: 'text' } },
+    touched: { control: { type: 'boolean' } },
+    'aria-label': { control: { type: 'text' } },
+    'aria-labelledby': { control: { type: 'text' } },
+    backspaceRemovesValue: { control: { type: 'boolean' } },
+    containerId: { control: { type: 'text' } },
+    isDisabled: { control: { type: 'boolean' } },
+    placeholder: { control: { type: 'text' } },
+    title: { control: { type: 'text' } },
+    isSearchable: { control: { type: 'boolean' } },
+    isClearable: { control: { type: 'boolean' } },
+    tabIndex: { control: { type: 'text' } },
+    tabSelectsValue: { control: { type: 'boolean' } },
+    cacheOptions: { control: { type: 'boolean' } },
+    allowCreateWhileLoading: { control: { type: 'boolean' } },
+    createOptionPosition: { control: 'select', options: ['first', 'last'] },
+    menuShouldBlockScroll: { control: { type: 'boolean' } },
+  },
+};
+export default meta;
+
+const animalOptions = [
```

Comment:
I think having this twice is fine, but if we find more uses for test data with this shape it might be worth making some sort of `utils` folder so it's not copy/pasted over and over.

Reply:
No question about it. I'd actually appreciate if we could use smth. like [faker.js](https://fakerjs.dev/) with a [seed](https://fakerjs.dev/api/randomizer.html#seed) in the future.

Reply:
I think it's worth bringing faker in for generating documentation data, but we should probably do so in a single effort instead of piecemeal?